### DISCUSSION TITLE
Include elevation in dxf file

### DISCUSSION THREAD

**Discussion by agustincaballero** [karttapullautin/karttapullautin#37] [Ideas]
Hi, I am from the mapant.es team.

I have a suggestion: to include the elevation in contours in file out2.dxf. This could be used for example in case you only want 10m contour lines (typically for MTB-O maps). Also it would enable us to label contour lines with elevation.

This can be done by adding a couple of lines and modifying another one when generating POLYLINE, like this:

out.push_str(format!("POLYLINE
 66
1
  8
{}
 **38
{}**
  0
", layer**, h**).as_str());

Hope this can be done; thanks in advance.

**Comment by rphlo** [comment]
Do you want to propose a PR?

**Comment by rphlo** [comment]
Thanks @agustincaballero for the PR.
